### PR TITLE
Fix javadoc build errors for Java 8

### PR DESCRIPTION
**Pull request by ricellis**
*What*
Resolve doclint javadoc validation errors so java-cloudant can build on Java 8 with default settings

*How*
Removed invalid javadoc to pass Java 8 doclint validation
Fixed codestyle to not insert invalid <p/> elements during formatting.

*Testing*
Clean gradle builds with javadoc on Java 8

reviewer @rhyshort 
reviewer @tomblench

### CODE REVIEW

**Comment by rhyshort** [comment]
+1

File: src/main/java/com/cloudant/client/api/CloudantClient.java [hunk 1]
```diff
@@ -48,9 +48,7 @@
  * <pre>
  * CloudantClient client = new CloudantClient("mycloudantaccount","myusername","mypassword");
  * </pre>
- * <p/>
```
Comment (tomblench) [inline src/main/java/com/cloudant/client/api/CloudantClient.java]: I think you need to fix up the paragraph tags throughout this javadoc block as they don't balance even after your changes. If it's a set of steps the user needs to go through then maybe a list (`<ul>`) is better.

Comment (tomblench) [inline src/main/java/com/cloudant/client/api/CloudantClient.java]: In general a bit more formatting here would make the output a bit nicer - I know you don't have time to fix all of the doc formatting but this is an important class.

Comment (ricellis) [inline src/main/java/com/cloudant/client/api/CloudantClient.java]: @tomblench - I've tried to make this better in 6f1350f, wdyt?

**Comment by tomblench** [comment]
Much nicer formatting +1

**Pull request merged**